kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExport.kt

```diff
@@ -135,6 +135,8 @@ internal class ObjCExport(val context: Context, symbolTable: SymbolTable) {
             |
             |    export *
             |    module * { export * }
+            |
+            |    use "Foundation"
```

Comment:
Are quotes required? The provided [documentation](https://clang.llvm.org/docs/Modules.html#use-declaration) doesn't use them. Are there examples of using this directive in the wild?

Reply:
I mean, if quotes are optional, I'm totally fine with keeping them. Please read my concern as "are they allowed by the specification and the implementation?"

Reply:
In practice clang seems to accept both forms:
`ModuleMapParser`'s `parseUseDecl` calls `parseModuleId` which consumes either an identifier or a string literal:
https://clang.llvm.org/doxygen/ModuleMap_8cpp_source.html

I agree that following the documentation more closely is better though, thanks for pointing this out.